apps/namadillo/src/App/Common/AppContainer.tsx

```diff
@@ -37,9 +33,6 @@ export const AppContainer = ({
               "flex items-center gap-3 text-yellow text-xl not-italic uppercase"
             )}
           >
-            <i className="w-[40px]">
-              <Logo eyeOpen={!hideBalance} />
```

Comment:
I think we should keep the logo, just setting the eyeOpen property as true

Reply:
Done!